assets/css/home.css

```diff
@@ -0,0 +1,660 @@
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 60px;
+}
+
+.all-rights {
+  display: flex;
+  background-color: #fff;
+  align-items: center;
+  padding: 1rem;
+  gap: 2rem;
+}
+
+.all-rights-detail {
+  display: none;
+}
+
+.all-rights-logo-mobile {
+  display: none;
+}
+
+.all-rights-logo {
+  width: 50%;
+}
+
+/* desktop version */
+@media only screen and (min-width: 768px) {
+  .set-background {
+    background-image: url("../images/bg/Hero-desktop.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    padding-block: 4rem;
+    opacity: 1;
+    padding-inline: 8vw;
+  }
+
+  .hamburger-img {
+    display: none;
+  }
+
+  .exit {
+    display: none;
+  }
+
+  .social-media {
+    background-color: var(--dark-color);
+    color: var(--primary-white);
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    position: fixed;
+    width: 100vw;
+    align-items: center;
+    align-content: center;
+    margin-top: -70px;
+    padding: 0 1%;
+    margin-left: -180px;
+  }
+
+  .social-media ul {
+    display: flex;
+    list-style-type: none;
+    align-items: center;
+    gap: 1px;
+    padding: 0 1%;
+  }
+
+  .nav-bar-ul li {
+    display: flex;
+    flex-direction: row;
+    width: auto;
+    height: auto;
+    position: initial;
+  }
+
+  .nav-bar-ul a {
+    font-family: "lato", sans-serif;
+  }
+
+  .social-media ul li {
+    list-style-type: none;
+    align-items: center;
+    padding: 0 1rem;
+  }
+
+  .social-media ul li a {
+    color: var(--secondary-white);
+    text-decoration: none;
+    font-size: 15px;
+  }
+
+  .social-media img {
+    width: 22px;
+    height: 22px;
+  }
+
+  .header {
+    padding: 0;
+    margin: 0;
+    width: 100vw;
+    display: flex;
+    justify-content: space-around;
+    position: fixed;
+  }
+
+  .nav-bar {
+    all: unset;
+    width: 100vw;
+    position: fixed;
+    margin-top: -18px;
+    background-color: #fff;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 0%;
+    margin-left: -173px;
+  }
+
+  .nav-bar-ul {
+    display: flex;
+    flex-direction: row;
+    width: auto;
+    height: auto;
+    font-size: 1rem;
+    position: initial;
+    margin: 2%;
+  }
+
+  #un-camp {
+    border: 6px solid var(--main-color);
+    color: var(--orange);
+    padding: 1px;
+    font-size: 20px;
+    margin-top: -5%;
+    margin-left: 0.8rem;
+  }
+
+  .my-logo {
+    display: block;
+  }
+
+  .Hero {
+    margin: 2rem auto 2rem 5rem;
+    width: 78%;
+  }
+
+  .underline {
+    height: 1px;
+    width: 10%;
+    margin: 1rem auto;
+    background-color: var(--main-color);
+  }
+
+  .program_details {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .program_card {
+    display: grid;
+    grid-template-columns: 10rem;
+    grid-template-rows: 35% 10% auto;
+    justify-items: center;
+  }
```

Comment:
- [ ] Please adjust the font size and spacing of your program section desktop version to give the content more space and make sure they don't collapse with each other just like in the provided design. Example snippet:

```suggestion
  .program_card {
    display: grid;
    grid-template-columns: 10rem;
    grid-template-rows: 35% 10% auto;
    justify-items: center;
    align-content: baseline;
    text-align: center;
    font-size: 0.85rem;
    padding: 1rem;
  }
```

Your design| Cindy Shin's design
|---|---|
|<img width="913" alt="Screenshot 2023-06-30 at 9 01 23 AM" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/44798044/b5c2df87-d7aa-448f-b766-78189c872c30">|<img width="870" alt="Screenshot 2023-06-30 at 9 01 34 AM" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/44798044/954145de-7965-4e95-9ef6-a9f6ef21346d">|